test/data/execute_2/manage/subcommand.py

```diff
@@ -0,0 +1,12 @@
+from argcmdr import cmd
+
+from .main import main
+
+import test_argcmdr
+
+
+@main.register
+@cmd('target')
+def subcommand(args):
+    target = getattr(test_argcmdr, args.target)
+    target.declare_success()
```

Comment:
It wasn't totally clear on first read what these test/data files were and how they fit in.  Looking at the `test_argcmdr` file helped, but just adding a bit of context as to how these commands should be used would be helpful when reading.

Reply:
I can understand how the diff might've been unclear, but I'm not entirely sure what you mean beyond that. These are just test fixtures, under the path `test/data/`. Do you mean a comment to reinforce this in each of these files?

Reply:
I think of fixtures as being data, not code. Not that there's a problem with this, but code fixtures are more opinionated about the context in which they are run; of course, data fixtures being plopped in a directory without context is its own problem, and generally leads to them being deprecated without anybody knowing.

In this case, it appears that it needs to specifically be called by a subclass of `TryExecuteTestCase` as the target, but the import and getattr implies a more generic usability.

Reply:
That makes sense.